_posts/2019-12-20-performance-now-conference-2019.md

```diff
@@ -0,0 +1,96 @@
+---
+layout:     post
+title:      Performance.Now() Conference 2019.
```

Comment:
Consistent formatting throughout the article - should be lower case even in titles, and with the parentheses, so `performance.now()`

Reply:
fixed